OutOfSchool/OutOfSchool.WebApi/Services/Images/ImageDependentEntityImagesInteractionService.cs

```diff
@@ -369,7 +369,11 @@ protected virtual bool AllowedToUploadGivenAmountOfFiles(TEntity entity, int cou
 
                 await ImageService.RemoveManyImagesAsync(imageIds).ConfigureAwait(false);
 
-                var imagesRemovingResult = new MultipleImageRemovingResult();
+                var imagesRemovingResult = new MultipleImageRemovingResult
```

Comment:
This looks like `MultipleImageRemovingResult` ctor responsibility. 
I mean next rule: constructor should create valid object; if object is invalid after constructing and you need to set some properties to make it valid - you have issues with constructing.